contracts/0.4.24/test_helpers/TestLido.sol

```diff
@@ -50,4 +50,20 @@ contract TestLido is Lido {
     function toLittleEndian64(uint256 _value) public pure returns (uint256 result) {
         return _toLittleEndian64(_value);
     }
+
+    /**
+    * @dev Public wrapper of internal fun. Internal function sets the address of Deposit contract
+    * @param _contract the address of Deposit contract
+    */
+    function setValidatorRegistrationContract(IValidatorRegistration _contract) public {
```

Comment:
Note: rename to `...DepositContract` after another PR with renaming will be merged